test/spec/ol/transform.test.js

```diff
@@ -0,0 +1,86 @@
+goog.provide('ol.test.transform');
+
+describe('ol.transform', function() {
+
+  function assertRoughlyEqual(t1, t2) {
+    t1.forEach(function(item, index) {
+      expect(item).to.roughlyEqual(t2[index], 1e-8);
+    });
+  }
+
+  describe('ol.transform.create()', function() {
+    it('creates an identity transform', function() {
+      expect(ol.transform.create()).to.eql([1, 0, 0, 1, 0, 0]);
+    });
+  });
+
+  describe('ol.transform.reset()', function() {
+    it('resets tansform to an identity transform', function() {
+      var transform = [1, 2, 3, 4, 5, 6];
+      expect(ol.transform.reset(transform)).to.eql([1, 0, 0, 1, 0, 0]);
+      expect(transform).to.eql([1, 0, 0, 1, 0, 0]);
+    });
+  });
+
+  describe('ol.transform.set()', function() {
+    it('sets the given values', function() {
+      var transform = ol.transform.create();
+      expect(ol.transform.set(transform, 1, 2, 3, 4, 5, 6)).to.eql([1, 2, 3, 4, 5, 6]);
+      expect(transform).to.eql([1, 2, 3, 4, 5, 6]);
+    });
+  });
+
+  describe('ol.transform.setFromArray()', function() {
+    it('sets values of 2nd transform on 1st transform', function() {
+      var transform1 = ol.transform.create();
+      var transform2 = [1, 2, 3, 4, 5, 6];
+      expect(ol.transform.setFromArray(transform1, transform2)).to.eql(transform2);
+      expect(transform1).to.eql(transform2);
+    });
+  });
+
+  describe('ol.transform.translate()', function() {
+    var transform = ol.transform.create();
+    expect(ol.transform.translate(transform, 3, 4)).to.eql([1, 0, 0, 1, 3, 4]);
+    expect(transform).to.eql([1, 0, 0, 1, 3, 4]);
```

Comment:
So that mocha can handle failure, these assertions (and the ones below) need to be in `it` functions.

See https://github.com/tschaub/ol3/commit/9a3e150acf2f0db74672674a9f6470e663deaae2 for a change that does this.